pandas/util/_print_versions.py

```diff
@@ -107,7 +107,7 @@ def show_versions(as_json: Union[str, bool] = False) -> None:
         j = {"system": sys_info, "dependencies": deps}
 
         if as_json is True:
-            print(j)
+            print(json.dumps(j))
```

Comment:
when printing to file, `json.dump(j, f, indent=2)` is used. would it be better to have the console output and file output the same?

maybe change the test, to ensure this consistency.